sdk/cargo-build-sbf/src/main.rs

```diff
@@ -1048,6 +1066,13 @@ fn main() {
                 .takes_value(false)
                 .conflicts_with("force_tools_install")
                 .help("Skip downloading and installing platform-tools, assuming they are properly mounted"),
+            )
+            .arg(
+                Arg::new("no_rustup")
+                .long("no-rustup")
```

Comment:
nit: since the `+solana` argument is known as a [toolchain override](https://rust-lang.github.io/rustup/overrides.html#toolchain-override-shorthand), can this be a bit more verbose, ie. `--no-rustup-override`?